docs/source/cpp/driver/postgresql.rst

```diff
@@ -47,17 +54,17 @@ the :cpp:class:`AdbcDatabase`.  This should be a `connection URI
          // Ignoring error handling
          struct AdbcDatabase database;
          AdbcDatabaseNew(&database, nullptr);
-         AdbcDatabaseSetOption(&database, "uri", "postgresql://localhost:5433", nullptr);
+         AdbcDatabaseSetOption(&database, "uri", "postgres://localhost:5433", nullptr);
```

Comment:
We can use `postgresql://` or `postgres://` here but how about using `postgresql://` here for consistency?

https://www.postgresql.org/docs/current/libpq-connect.html#id-1.7.3.8.3.6

> The URI scheme designator can be either postgresql:// or postgres://. Each of the remaining URI parts is optional.